### ISSUE TITLE
alternatives to openpose for demo

### ISSUE THREAD

**Issue by sophia-wright-blue** [yysijie/st-gcn#95]
### Code version (Git Hash) and PyTorch version

PyTorch: 0.4.0

### Dataset used

Kinetics Skeleton

### Comments

to run the demo, instead of using the caffe Openpose implementation, would it be possible to use a tensor flow or pytorch openpose implementation such as here: https://github.com/ildoonet/tf-pose-estimation

If yes, what changes would I need to make to st-gcn code to run the demo?

Thanks,

===============

**Comment by yysijie** [comment]
Hi, @sophia-wright-blue,
Please refer to https://github.com/yysijie/st-gcn/blob/master/processor/demo.py.
This script extracts human skeletons from video frames by Caffe Openpose and transforms them into the format of our deep network input.
You should modify these two parts.

**Comment by sophia-wright-blue** [comment]
Thank you for replying @yysijie , I will modify the demo.py script and try other openpose implementations. if you can refer me to some TensorFlow or pytorch implementations of openpose that are good substitutes, that'd be very helpful, thanks,

**Comment by huanglai666** [comment]
> Thank you for replying @yysijie , I will modify the demo.py script and try other openpose implementations. if you can refer me to some TensorFlow or pytorch implementations of openpose that are good substitutes, that'd be very helpful, thanks,

@sophia-wright-blue hi，I am doing the same job, do you implement the replacement of openpose, or use the other pose estimation methods to achieve the same accuracy? If we can talk about it, it will be very helpful to me.

**Comment by sophia-wright-blue** [comment]
Hi @huanglai666 , thanks for reaching out, unfortunately, I have moved on to other projects, so was not able to spend more time on this repo, good luck to you!

**Comment by CompliceU** [comment]
Hello, these two days are also trying to use the tf-pose to replace openpose, I do not know if you can complete this work, I hope to guide, thank you @sophia-wright-blue @huanglai666

**Comment by sophia-wright-blue** [comment]
Hi @CompliceU , I wish I could help, but I'm busy with some other projects, good luck!

**Comment by CompliceU** [comment]
Thank you very much for your reply. I don't know if you can take the time to give me a little guidance. I really want to replace openpose and show this effect in real time.




------------------ 原始邮件 ------------------
发件人: "sophia-wright-blue"<notifications@github.com>;
发送时间: 2019年6月11日(星期二) 晚上9:58
收件人: "yysijie/st-gcn"<st-gcn@noreply.github.com>;
抄送: "蠢货"<1215533585@qq.com>;"Mention"<mention@noreply.github.com>;
主题: Re: [yysijie/st-gcn] alternatives to openpose for demo (#95)




Hi @CompliceU , I wish I could help, but I'm busy with some other projects, good luck!

**Comment by sophia-wright-blue** [comment]
I'm terribly sorry, @CompliceU :( I wish I could, but I'm swamped right now, good luck!
